[2025-06-09T07:59:05.331+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-09T07:59:05.342+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stephen_curry_stats_medallion_etl.ensure_bucket_exists manual__2025-06-09T07:59:04.757055+00:00 [queued]>
[2025-06-09T07:59:05.346+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stephen_curry_stats_medallion_etl.ensure_bucket_exists manual__2025-06-09T07:59:04.757055+00:00 [queued]>
[2025-06-09T07:59:05.346+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-06-09T07:59:05.351+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): ensure_bucket_exists> on 2025-06-09 07:59:04.757055+00:00
[2025-06-09T07:59:05.354+0000] {standard_task_runner.py:72} INFO - Started process 366 to run task
[2025-06-09T07:59:05.356+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stephen_curry_stats_medallion_etl', 'ensure_bucket_exists', 'manual__2025-06-09T07:59:04.757055+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/medallion_sales_etl_dag.py', '--cfg-path', '/tmp/tmprsn46crz']
[2025-06-09T07:59:05.358+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask ensure_bucket_exists
[2025-06-09T07:59:05.386+0000] {task_command.py:467} INFO - Running <TaskInstance: stephen_curry_stats_medallion_etl.ensure_bucket_exists manual__2025-06-09T07:59:04.757055+00:00 [running]> on host 8fa8a988967d
[2025-06-09T07:59:05.435+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stephen_curry_stats_medallion_etl' AIRFLOW_CTX_TASK_ID='ensure_bucket_exists' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T07:59:04.757055+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T07:59:04.757055+00:00'
[2025-06-09T07:59:05.435+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-09T07:59:05.445+0000] {logging_mixin.py:190} INFO - Bucket 'mybucket' already exists.
[2025-06-09T07:59:05.445+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-06-09T07:59:05.460+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-09T07:59:05.460+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stephen_curry_stats_medallion_etl, task_id=ensure_bucket_exists, run_id=manual__2025-06-09T07:59:04.757055+00:00, execution_date=20250609T075904, start_date=20250609T075905, end_date=20250609T075905
[2025-06-09T07:59:05.497+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-09T07:59:05.512+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-09T07:59:05.513+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
